HW_4/Task1.py

```diff
@@ -0,0 +1,12 @@
+# Напишите функцию для транспонирования матрицы
+
+import numpy as np
+
+
+def transposition(matrix):
+    transpose_matrix = matrix.transpose()
+    return transpose_matrix
```

Comment:
Вообще можно обойтись средствами встроенных функций, стандартная библиотека тут даже не нужна
```suggestion
def transposition(matrix):
    return [*map(list, zip(*matrix))]
```